orangecontrib/text/widgets/owpreprocess.py

```diff
@@ -960,6 +960,10 @@ def __init__(self):
         self.preview = ""
         gui.label(box, self, "%(preview)s", wordWrap=True)
         self.controlArea.layout().insertWidget(1, box)
+        box = gui.vBox(self.controlArea, "Output")
+        self.output_info = ""
+        gui.label(box, self, "%(output_info)s", wordWrap=True)
+        self.controlArea.layout().insertWidget(2, box)
         self.controlArea.setFixedWidth(220)
```

Comment:
While you're at it, please remove this line. The control area will look a lot better.